spec/controllers/ops_controller/settings/upload_spec.rb

```diff
@@ -11,58 +11,49 @@
     end
 
     it 'adds success flash message to @flash_array regarding updating custom logo image with a .png file' do
-      file = 'app/assets/images/layout/login-screen-logo.png'
-      controller.params = {:upload => {:logo => ActionDispatch::Http::UploadedFile.new(:tempfile => File.open(file), :filename => 'login-screen-logo.png', :type => 'image/png')}}
+      controller.params = {:upload => {:logo => fixture_file_upload('spec/fixtures/image.png')}}
       controller.send(:upload_logo)
-      expect(controller.instance_variable_get(:@flash_array)).to include(:message => 'Custom logo image "login-screen-logo.png" uploaded', :level => :success)
+      expect(controller.instance_variable_get(:@flash_array)).to include(:message => 'Custom logo image "image.png" uploaded', :level => :success)
     end
 
     it 'adds error flash message to @flash_array regarding updating custom logo image with a non-png file' do
-      file = File.join(__dir__, '/data/test.txt.png')
-      controller.params = {:upload => {:logo => ActionDispatch::Http::UploadedFile.new(:tempfile => File.open(file), :filename => 'test.txt.png', :type => 'image/png')}}
+      controller.params = {:upload => {:logo => fixture_file_upload('spec/fixtures/test.txt.png')}}
```

Comment:
Why was the mime-type not passed to `fixture_file_upload` ?  I'm wondering if that's important for the test, particularly where we're trying to test hacking around file detection like in this case.

Reply:
added mime types to all requests